apps/dsp-app/src/app/main/action/login-form/login-form.component.ts

```diff
@@ -142,16 +150,19 @@ export class LoginFormComponent implements OnInit, OnDestroy {
         next: loginResult => {
           if (loginResult) {
             this._componentCommsService.emit(new EmitEvent(Events.loginSuccess, true));
-
-            return this._authService
-              .loadUser(identifier)
+            this._store.dispatch(new LoadUserAction(identifier));
+            return combineLatest([
```

Comment:
What about using this.store.dispatch(new LoadUserAction(identifier)).pipe()

rather than combineLatest ?